<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWindowModel</class>
 <widget class="QDialog" name="DataWindowModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>teacher edit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttons">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>19</y>
     <width>511</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="main">
    <item row="0" column="0">
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="5">
     <widget class="QLineEdit" name="name_edit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="teacher_label">
      <property name="text">
       <string>teacher</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="5">
     <widget class="QLineEdit" name="teacher_edit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="background_color_label">
      <property name="text">
       <string>background color</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="red_background_color">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="green_background_color">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="blue_background_color">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="background_color_edit">
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="text_color_label">
      <property name="text">
       <string>text color</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="red_text_color">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="green_text_color">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="blue_text_color">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="text_color_edit">
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="text_color_prevew">
      <property name="text">
       <string>       </string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="background_color_prevew">
      <property name="text">
       <string>       </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttons</tabstop>
  <tabstop>name_edit</tabstop>
  <tabstop>teacher_edit</tabstop>
  <tabstop>red_background_color</tabstop>
  <tabstop>green_background_color</tabstop>
  <tabstop>blue_background_color</tabstop>
  <tabstop>background_color_edit</tabstop>
  <tabstop>red_text_color</tabstop>
  <tabstop>green_text_color</tabstop>
  <tabstop>blue_text_color</tabstop>
  <tabstop>text_color_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>DataWindowModel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>DataWindowModel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
